### PR TITLE
Display rebuild link in local case only #286

### PR DESCRIPTION
**Pull request by YeeSkywalker**
Date: Mar 2nd
Author: Yee Lin
Issue: [#286](https://github.com/adoptium/TKG/issues/286)

### CODE REVIEW

File: scripts/resultsSum.pl [hunk 1]
```diff
@@ -260,47 +260,47 @@ sub resultReporter {
...
+			}
+			if ( defined $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'} ) {
+				$jenkinsParam .= "&CUSTOMIZED_SDK_URL_CREDENTIAL_ID=" . $ENV{'CUSTOMIZED_SDK_URL_CREDENTIAL_ID'};
+			}
+			if ( defined $ENV{'UPSTREAM_JOB_NAME'} ) {
+				$jenkinsParam .= "&UPSTREAM_JOB_NAME="   . $ENV{'UPSTREAM_JOB_NAME'};
+			}
+			if ( defined $ENV{'UPSTREAM_JOB_NUMBER'} ) {
+				$jenkinsParam .= "&UPSTREAM_JOB_NUMBER=" . $ENV{'UPSTREAM_JOB_NUMBER'};
+			}
```
Comment (llxia) [inline scripts/resultsSum.pl:287]: We can remove Line 267 - 287.

Comment (YeeSkywalker) [reply scripts/resultsSum.pl]: Hi @llxia, I have removed line 267 - 287, please review when you are free.

**Comment by llxia** [comment]
Please run Grinder to test your PR: one with passed test and one with a failed test. Thanks

**Comment by llxia** [comment]
https://github.com/eclipse-openj9/openj9/wiki/Reproducing-Test-Failures-Locally

**Comment by smlambert** [comment]
So, from that link that Lan shared, detailed instructions here:

1. Ensure you have the [test prereqs](https://github.com/adoptium/aqa-tests/blob/master/doc/Prerequisites.md) 
1. Download a jdk8 build for mac, here is the [link to the latest released .pkg](https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u322-b06/OpenJDK8U-jdk_x64_mac_hotspot_8u322b06.pkg) from `https://adoptium.net/archive.html?variant=openjdk8&jvmVariant=hotspot` When you install it, copy the path it ends up getting installed to, it'll be something like: /Library/Java/JavaVirtualMachines/temurin-8.jdk (which you need in next step)
2. export TEST_JDK_HOME=/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home
3. git clone https://github.com/adoptium/aqa-tests.git
4. cd aqa-tests
5. ./get.sh --tkg_repo https://github.com/YeeSkywalker/TKG.git --tkg_branch #286
6. cd TKG
7. export BUILD_LIST=openjdk
8. make _jdk_jdi_jdk8_0

**Comment by YeeSkywalker** [comment]
> So, from that link that Lan shared, detailed instructions here:
> 
> 1. Ensure you have the [test prereqs](https://github.com/adoptium/aqa-tests/blob/master/doc/Prerequisites.md)
> 2. Download a jdk8 build for mac, here is the [link to the latest released .pkg](https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u322-b06/OpenJDK8U-jdk_x64_mac_hotspot_8u322b06.pkg) from `https://adoptium.net/archive.html?variant=openjdk8&jvmVariant=hotspot` When you install it, copy the path it ends up getting installed to, it'll be something like: /Library/Java/JavaVirtualMachines/temurin-8.jdk (which you need in next step)
> 3. `export TEST_JDK_HOME=/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home`
> 4. `git clone https://github.com/adoptium/aqa-tests.git`
> 5. `cd aqa-tests`
> 6. `./get.sh --tkg_repo https://github.com/YeeSkywalker/TKG.git --tkg_branch #286`
> 7. `cd TKG`
> 8. `export BUILD_LIST=openjdk`
> 9. `make _jdk_jdi_jdk8_0`

I followed the all intruction above and got an Error `Could not find or load main class org.testKitGen.MainRunner`. I am wondering what is this standing for and how could I solve it? Thank you for your time in advance!
Screen Shot 2022-03-23 at 2 18 43 AM

**Comment by YeeSkywalker** [comment]
> So, from that link that Lan shared, detailed instructions here:
> 
> 1. Ensure you have the [test prereqs](https://github.com/adoptium/aqa-tests/blob/master/doc/Prerequisites.md)
> 2. Download a jdk8 build for mac, here is the [link to the latest released .pkg](https://github.com/adoptium/temurin8-binaries/releases/download/jdk8u322-b06/OpenJDK8U-jdk_x64_mac_hotspot_8u322b06.pkg) from `https://adoptium.net/archive.html?variant=openjdk8&jvmVariant=hotspot` When you install it, copy the path it ends up getting installed to, it'll be something like: /Library/Java/JavaVirtualMachines/temurin-8.jdk (which you need in next step)
> 3. `export TEST_JDK_HOME=/Library/Java/JavaVirtualMachines/temurin-8.jdk/Contents/Home`
> 4. `git clone https://github.com/adoptium/aqa-tests.git`
> 5. `cd aqa-tests`
> 6. `./get.sh --tkg_repo https://github.com/YeeSkywalker/TKG.git --tkg_branch #286`
> 7. `cd TKG`
> 8. `export BUILD_LIST=openjdk`
> 9. `make _jdk_jdi_jdk8_0`

By following the instruction, the local TKG test failed with output:  

```console
TEST RESULT: Failed. Execution failed: exit code 134
--------------------------------------------------
Test results: passed: 146; failed: 1
Report written to /Users/yeeskywalker/jvmtest/openjdk/report/html/report.html
Results written to /Users/yeeskywalker/aqa-tests/TKG/output_16482845158910/jdk_jdi_jdk8_0/work
Error: Some tests failed or other problems occurred.

jdk_jdi_jdk8_0_FAILED

Nothing to be done for teardown.
jdk_jdi_jdk8_0 Finish Time: Sat Mar 26 02:50:09 2022 Epoch Time (ms): 1648284609980
cd "/Users/yeeskywalker/aqa-tests/TKG/../TKG/scripts"; \
	perl "resultsSum.pl" --failuremk="/Users/yeeskywalker/aqa-tests/TKG/../TKG/failedtargets.mk" --resultFile="/Users/yeeskywalker/aqa-tests/TKG/../TKG/output_16482845158910/TestTargetResult" --platFile="/Users/yeeskywalker/aqa-tests/TKG/../TKG/resources/buildPlatformMap.properties" --diagnostic=failure --jdkVersion=8 --jdkImpl=hotspot --jdkVendor="Temurin" --spec=osx_x86-64 --buildList=openjdk --customTarget= --testTarget=jdk_jdi_jdk8_0 --tapPath=/Users/yeeskywalker/aqa-tests/TKG/../TKG/output_16482845158910/ --tapName=
Global symbol "$rebuildLinkBase" requires explicit package name (did you forget to declare "my $rebuildLinkBase"?) at resultsSum.pl line 268.
Global symbol "$rebuildLinkBase" requires explicit package name (did you forget to declare "my $rebuildLinkBase"?) at resultsSum.pl line 272.
Global symbol "$rebuildLinkBase" requires explicit package name (did you forget to declare "my $rebuildLinkBase"?) at resultsSum.pl line 277.
Execution of resultsSum.pl aborted due to compilation errors.
make[1]: *** [resultsSummary] Error 255
make: *** [_jdk_jdi_jdk8_0] Error 2
```

**Comment by YeeSkywalker** [comment]
> @YeeSkywalker any update on this issue? Thanks

Update test shortly!

**Comment by YeeSkywalker** [comment]
In order to avoid merge conflict, move this PR to [PR/308](https://github.com/adoptium/TKG/pull/308)

**Pull request closed without merging**